### ISSUE TITLE
【建议】希望集成Spring体系的MircoMeter运行指标

### ISSUE THREAD

**Issue by dargoner** [open-job/openjob#136]
集成Spring体系的MircoMeter，使得OpenJob的运行时Metrics可以被监控起来。Powerjob还没实现

**Comment by stelin** [comment]
@dargoner 我们评估下

**Comment by 172151295** [comment]
> @dargoner 我们评估下

什么时候发布下一个版本@stelin

**Comment by Leowuqunqun** [comment]
@stelin assign me

**Comment by Leowuqunqun** [comment]
@172151295 
The metrics API integrated in Spring Boot's dependency actuator uses the framework Micrometer. 
I'm exposing the metrics through actuator and it seems to be working, any questions you can discuss with me

**Comment by dargoner** [comment]
这是一个亮点特性啊

**Comment by Leowuqunqun** [comment]
> 这是一个亮点特性啊
@dargoner 
Yes,I'm working on this task, integrating metrics and building a dashboard using Prometheus and Grafana, of course I know there are more options for micrometer, I'm just providing a sample, you can see the sample in the documentation in the next release!

**Comment by xinzhuxiansheng** [comment]
@Leowuqunqun  Hi, you can try `jmx_prometheus_javaagent`. For Prometheus it is more useful.
